nomad_darwin.go

```diff
@@ -31,10 +32,17 @@ func init() {
 }
 
 func readDefaultForNoMAD(key string) (string, error) {
-	cmd := execCommand("defaults", "read", "com.trusourcelabs.NoMAD", key)
-	out, err := cmd.Output()
+	userDomain := "com.trusourcelabs.NoMAD"
+	mpDomain := fmt.Sprintf("/Library/Managed Preferences/%s.plist", userDomain)
+
+	// Read from managed preferences first
+	out, err := execCommand("defaults", "read", mpDomain, key).Output()
+	if err != nil {
+			// Read from user preferences if not in managed preferences
+			out, err = execCommand("defaults", "read", userDomain, key).Output()
+	}
 	if err != nil {
-		return "", err
+			return "", err
```

Comment:
This and some previous lines are indented too far. This means you have not run `go fmt` on the file. If you want to do more Go development I recommend setting up your editor to run it every time you save. Even better is to install `goimports` (https://godoc.org/golang.org/x/tools/cmd/goimports) which does all that `go fmt` does but also adds and removes required imports. But if this is just a one-off Go exercise for you, just run `go fmt` over the file (`go help fmt` for more details).